### PR TITLE
feat(api,ui): run v2 show repo info

### PR DESCRIPTION
**Pull request by richardlt**
1. Description
1. Related issues
1. About tests
1. Mentions

@ovh/cds

### CODE REVIEW

File: engine/api/workflow_v2/dao_run.go [hunk 1]
```diff
@@ -308,13 +334,22 @@ func SearchRuns(ctx context.Context, db gorp.SqlExecutor, projKey string, filter
     SELECT *
     FROM v2_workflow_run
     WHERE 
-			project_key = $1 
+			project_key = $1
```
Comment (sguiheux) [inline engine/api/workflow_v2/dao_run.go:337]: Index for this kind of request?

**Review by sguiheux — CHANGES_REQUESTED** [review]

**Review by sguiheux — APPROVED** [review]

**Comment by ovh-cds** [comment]
CDS Report build-cds#19970.0 ✔ 
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✔ 
  * Test SDK ✔ 
  * Test UI ✔

**Comment by ovh-cds** [comment]
CDS Report integration-cds#19970.0 ✘  
* Integration
  * Docker Compose Tests ✔ 
  * Standalone Tests ✔ 
  * Standalone Tests v1 ✔ 
  * Standalone Tests v2 ✘

**Comment by ovh-cds** [comment]
CDS Report build-cds#19974.0 ✘  
* Build
  * Build and Package All ✘  
  * Build Debpacker ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✔ 
  * Test SDK ✔ 
  * Test UI ✔

**Comment by ovh-cds** [comment]
CDS Report build-cds#19975.0 ✔ 
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✔ 
  * Test SDK ✔ 
  * Test UI ✔

**Comment by ovh-cds** [comment]
CDS Report integration-cds#19975.0 ✔ 
* Integration
  * Docker Compose Tests ✔ 
  * Standalone Tests ✔ 
  * Standalone Tests v1 ✔ 
  * Standalone Tests v2 ✔

**Pull request merged**